WordPressAuthenticator/GoogleSignIn/JWToken.swift

```diff
@@ -0,0 +1,47 @@
+/// Represents a JSON Web Token (JWT)
+///
+/// See https://jwt.io/introduction
+struct JWToken {
```

Comment:
What do you think about using the full name as the type name here, like `JSONWebToken`?

Reply:
Great suggestion. It's a bit inconsistent as J (short) W (short) Token (long).

The alternative would be `JWT`, similar to `URL`. But I've gone with the more verbose option you suggested in 2e99cef925a2c6aa42ff13db37e205ab2a704c1b as it seems clearer.